libs/cardano-ledger-core/src/Cardano/Ledger/Plutus/TxInfo.hs

```diff
@@ -116,7 +116,7 @@ txOutSourceToText = \case
   TxOutFromOutput txIx -> "Output: " <> T.pack (show txIx)
 
 transBoundedRational :: BoundedRational r => r -> PV3.Rational
-transBoundedRational = PV3.fromGHC . unboundRational
+transBoundedRational = PV3.fromHaskellRatio . unboundRational
```

Comment:
@bezirg, which version of `plutus-ledger-api` started exporting `fromHaskellRatio`? We need to add a lower bound in the cabal file on that version in order to make sure cabal doesn't try to build an incompatible version.